### ISSUE TITLE
Marker is not a chunk.

### ISSUE THREAD

**Issue by Naphho** [TownyAdvanced/Pl3xMap-Towny#14]
## Describe the Bug
The marker is 15*15 blocks not a chunk.
![bug](https://user-images.githubusercontent.com/59295406/144170315-da57ea0b-457f-4c00-a98d-865b21384c42.png)

## Expected Behavior
The marker should be the red square which is a chunk.

### Plugin Version
1.0.0 with default config.

### Towny Version
0.97.3.0 with default config.

### Pl3xMap Version
1.0.0-BETA-162. with default config.

### Server Software Version
Paper 1.17.1.

**Comment by Naphho** [comment]
![fixed](https://user-images.githubusercontent.com/59295406/144755883-addeeeeb-acc5-4dba-a112-15ab93289204.png)

Anyway, I fixed it!

**Comment by silverwolfg11** [comment]
Oh, sorry for the late response, I've been busy recently. Glad you fixed it! If you're up to it, you can always make a pull-request with the fix. If not, I'll look into it when I time.

**Comment by silverwolfg11** [comment]
I tried to reproduce the issue but the marker size seems correct to me.

Steps for reproduction:
- Clean server with Pl3xMap, Towny, and Pl3xMap-Towny Install
- Go to chunk X: 6, Z: -14
- Look at Pl3xMap and use mouse to hover over coordinates (I had to zoom in with my browser to get the precision I wanted).

Doing that produced these results on the Pl3xMap:
![image](https://user-images.githubusercontent.com/16297359/146135680-4bb3052d-6109-4638-b159-29e17cff6784.png)
![image](https://user-images.githubusercontent.com/16297359/146135733-cf0119a2-5377-431a-94c2-0a6b038d4da7.png)

Using https://dinnerbone.com/minecraft/tools/coordinates/ and plugging in the previously mentioned X and Z coords for the chunk you'll see that the tool says "This chunk section contains blocks 96,0,-224 to 111,15,-209". The coordinates are a bit mis-matched, but they still represent the correct borders of the chunk indicating that the marker is the correct size.
